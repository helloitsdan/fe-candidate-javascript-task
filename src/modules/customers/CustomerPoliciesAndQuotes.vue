<template>
  <div class="background--barely">
    <section class="container padded-block">
      <policy-list />
    </section>
    <section class="container padded-block">
      <quote-list />
    </section>
  </div>
</template>

<script lang="ts">
import Vue from "vue";

import QuoteList from "@/modules/quotes";
import PolicyList from "@/modules/policies";

export default Vue.extend({
  name: "CustomerPoliciesAndQuotes",
  components: {
    QuoteList,
    PolicyList
  }
});
</script>
