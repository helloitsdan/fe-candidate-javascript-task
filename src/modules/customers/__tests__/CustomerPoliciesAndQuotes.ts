import { shallowMount } from "@vue/test-utils";

import CustomerPoliciesAndQuotes from "../CustomerPoliciesAndQuotes.vue";

describe("CustomerPoliciesAndQuotes", () => {
  it("renders", () => {
    const wrapper = shallowMount(CustomerPoliciesAndQuotes);
    expect(wrapper.element).toMatchSnapshot();
  });
});
