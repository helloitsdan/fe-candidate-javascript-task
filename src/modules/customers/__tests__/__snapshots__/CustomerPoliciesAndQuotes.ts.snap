// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomerPoliciesAndQuotes renders 1`] = `
<div
  class="background--barely"
>
  <section
    class="container padded-block"
  >
    <policy-list-stub />
  </section>
   
  <section
    class="container padded-block"
  >
    <quote-list-stub />
  </section>
</div>
`;
